packages/api/cms-api/src/user-permissions/content-scope.service.ts

```diff
@@ -105,6 +105,9 @@ export class ContentScopeService {
             return GqlExecutionContext.create(context).getArgs();
         } else {
             const request = context.switchToHttp().getRequest();
+            if (request.query?.scope) {
+                request.query.scope = JSON.parse(request.query.scope);
```

Comment:
This could in theory lead to internal server exceptions:

<img width="660" alt="image" src="https://github.com/user-attachments/assets/47530993-9d7a-466b-bdde-cd9888f11e5d">

Should we cover that?